modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-instance-aws-load-balancer-controller_{context}"]
+= Creating an instance of AWS Load Balancer Controller
+
+After installing the operator, you can create an instance of `AWSLoadBalancerController`. You can install only a single instance of the `aws-load-balancer-controller` in a cluster. The operator.only reconciles the resource with the name `cluster`.
+
+.Prerequisites
+
+.Procedure
+
+. Create the `AWSLoadBalancerController` resource:
++
+[source,yaml]
+----
+apiVersion: networking.olm.openshift.io/v1alpha1
+kind: AWSLoadBalancerController <1>
+metadata:
+  name: cluster <2>
```

Comment:
This is the name of the instance of the _aws-load-balancer-controller_ will be added as a suffix to all related resources.

Reply:
Addressed